fix: change ZeroOrTenDistrict rule to only check 10

ioannavlahou opened the pull request:
I think the rule is misinterpreted - although BS is the only area to have both a district 0 and a district 10, it is not the only area to have a district 10.

guimunarolo commented:
Hi @ioannavlahou!

I've fixed the CI and switched pipenv by poetry.
Could you merge with master, solve the conflicts?

ioannavlahou commented:
> Hi @ioannavlahou!
> 
> I've fixed the CI and switched pipenv by poetry. Could you merge with master, solve the conflicts?

Hi @guimunarolo - apologies for the embarrassingly long delay! I've merged master in and solved the conflicts.
